packages/apps/shopify-api/lib/clients/admin/graphql/client.ts

```diff
@@ -62,6 +62,7 @@ export class GraphqlClient {
       customFetchApi: abstractFetch,
       logger: clientLoggerFactory(config),
       userAgentPrefix: getUserAgent(config),
+      isTesting: config.isTesting,
```

Comment:
`config.isTesting` could be undefined here right? Should we default this to false?

Reply:
I think that should be fine! You can keep it like this.